recipes/python-lambda-4dn/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-lambda-4dn" %}
+{% set version = "0.12.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "a86d8b438e041cc7b0d144e16f24bae16cdbb4003ef2b55bbc89b187f2da7e0f"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - boto3 >=1.7.42
+    - botocore >=1.10.42
+    - docutils >=0.14
+    - pip
+    - python
+  run:
+    - boto3 >=1.7.42
+    - botocore >=1.10.42
+    - docutils >=0.14
+    - python
+
+test:
+  imports:
+    - aws_lambda
+    - aws_lambda.examples
+    - tests
+
+about:
+  home: "https://github.com/4dn-dcic/python-lambda"
+  license: "ISC (ISCL)"
+  license_family: "OTHER"
+  license_file: ""
+  summary: "FORKED for 4dn-dcic. Use to package and deploy lambda functions."
+  doc_url: ""
+  dev_url: ""
```

Comment:
Please add doc and dev urls here and for the other recipes.

Reply:
Since the github repo is already the homepage, you can also remove the dev_url line entirely.